webhooks.md

```diff
@@ -0,0 +1,114 @@
+# Webhooks
+
+Whenever integration client is changing its state webhook message is sent to certain webhook url field set on that client. Webhook url is instatiated per integration and responds on action made on property level.
+
```

Comment:
I would write here the list of actions
like webhook actions:
* created
* disabled, ...

and then the definition as you have.